metric/system/process/process_windows.go

```diff
@@ -199,6 +199,9 @@ func FillMetricsRequiringMoreAccess(pid int, state ProcState) (ProcState, error)
 }
 
 func getProcArgs(pid int) ([]string, error) {
+	if ok := shouldIgnore(pid); ok {
+		return []string{}, nil
```

Comment:
Do we need to create a non-nil empty slice? Can we just return `nil`?

https://go.dev/wiki/CodeReviewComments#declaring-empty-slices

Reply:
Oh, missed this. Returning `nil` is better.